### DISCUSSION TITLE
Wasp lang server for neovim

### DISCUSSION THREAD

**Discussion by shivamnarkar47** [wasp-lang/wasp#2197] [Archived]
Hey ! I think a LSP server for wasp must be available for neovim too. I prefer neovim as my standard editor and wasp lang doesn't give me much access to it 🫠. I request this feature .

**Comment by Martinsos** [comment]
Thanks for the request :D!
There is a lot of things we are wroking on and we haven't been able to prioritize this so far: I myself am on emacs and lot of team members are on vim btw. But, we have something else coming soon: ability to write `main.wasp` in TypeScript (`main.wasp.ts`), which will mean it will be well supported in any editor with good TS support.
That said, if you wanted proper support for wasp for neovim right now, nothing is stopping you from implementing your own extension for neovim that uses wasp's language server. Feel free to take a look at how we implemented vscode extension and draw inspiration from it!
